core/tests/protractor_utils/TopicViewerPage.js

```diff
@@ -58,6 +60,35 @@ var TopicViewerPage = function() {
       expect(await storySummaryTitleList.count()).toEqual(count);
     }
   };
+
+  this.moveToRevisionTab = async function() {
+    var revisionTabLink = element(by.css('.protractor-test-revision-tab-link'));
+    await action.click('Revision Tab', revisionTabLink);
+    await waitFor.pageToFullyLoad();
```

Comment:
Just to double-check, does the page show a loading indicator with the `.protractor-test-loading-fullpage` HTML class? That's all the `waitFor.pageToFullyLoad()` function checks for:

https://github.com/oppia/oppia/blob/ee6150bef1a3d0d0d0f1e1f1fe4d91d025b43383/core/tests/protractor_utils/waitFor.js#L61-L70

This goes for everywhere you use `waitFor.pageToFullyLoad()`

Reply:
The page does not show a loading indicator with the above HTML class, removed ```waitFor.pageToFullyLoad()```.